src/expr.rs

```diff
@@ -1860,7 +1860,10 @@ fn rewrite_let(
     // TODO(ytmimi) comments could appear between `let` and the `pat`
 
     // 4 = "let ".len()
-    let pat_shape = shape.offset_left(4)?;
+    let mut pat_shape = shape.offset_left(4)?;
+    if context.config.version() == Version::Two {
+        pat_shape = pat_shape.sub_width(2)?;
```

Comment:
Let's leave a comment explaining why we're subtracting 2 from the width. Something like `// 2 for the length of " ="` should do.